projects/m1/005-units-of-time-again/solutions/MatteoFaienza/units-of-time-again.py

```diff
@@ -0,0 +1,18 @@
+# input user seconds
+secondUser=int(input('Enter number of second '))
+# seconds in a day,hour and minute
+secDay=86400
+secHour=3600
+secMinut=60
```

Comment:
You should always try to use word naming that allows people to understand what's going on.

For example, in this case, a better naming would be: minutesToSeconds or secondsInAMinute